libqtile/widget/maildir.py

```diff
@@ -34,8 +34,8 @@ class Maildir(base.ThreadedPollText):
     """A simple widget showing the number of new mails in maildir mailboxes"""
     orientations = base.ORIENTATION_HORIZONTAL
     defaults = [
-        ("maildirPath", "~/Mail", "path to the Maildir folder"),
-        ("subFolders", [], 'The subfolders to scan (e.g. [{"path": "INBOX", '
+        ("maildir_path", "~/Mail", "path to the Maildir folder"),
+        ("sub_folders", [], 'The subfolders to scan (e.g. [{"path": "INBOX", '
```

Comment:
Yeah, I did not change those as we didn't want to break configs, but since we're breaking everything those days let's do it. Let's put a quick word about this into the Changelog though.